core/test/testmodelarraydetails/include/ModelArrayDetails.hpp

```diff
@@ -0,0 +1,31 @@
+/*!
+ * @file ModelArrayDetails.hpp
+ *
+ * @date Oct 19, 2022
+ * @author Tim Spain <timothy.spain@nersc.no>
+ */
+
+#ifndef MODELARRAYDETAILS_HPP
+#define MODELARRAYDETAILS_HPP
+
+    enum class Dimension {
+        X,
+        Y,
+        Z,
+        U,
+        COUNT
+    };
+
+    enum class Type {
+        ONED,
+        TWOD,
+        DOSD,
```

Comment:
Why ``DOSD``?

Reply:
I needed a second 'two d'. So I went with the Spanish for two, as I didn't want to call the dimension Tod.